test/e2e/tests/account/account-custom-name.spec.ts

```diff
@@ -14,11 +14,10 @@ describe('Account Custom Name Persistence', function (this: Suite) {
     await withFixtures(
       {
         fixtures: new FixtureBuilder().build(),
-        ganacheOptions: defaultGanacheOptions,
         title: this.test?.fullTitle(),
       },
       async ({ driver }: { driver: Driver }) => {
-        await loginWithBalanceValidation(driver);
+        await loginWithBalanceValidation(driver, '0');
```

Comment:
mmm I think we might want to tweak this function. Instead of having to pass manually the balance, we could use the same approach for the other loginWithBalanceValidation function we have, where we query directly ganache, so there's no need to pass any balance manually.
What do you think?

```
const logInWithBalanceValidation = async (driver, ganacheServer) => {
  await unlockWallet(driver);
  // Wait for balance to load
  await locateAccountBalanceDOM(driver, ganacheServer);
};
```
and here the implementation of locate balance using ganache:
```

const locateAccountBalanceDOM = async (
  driver,
  ganacheServer,
  address = null,
) => {
  const balanceSelector = '[data-testid="eth-overview__primary-currency"]';
  if (ganacheServer) {
    const balance = await ganacheServer.getBalance(address);
    await driver.waitForSelector({
      css: balanceSelector,
      text: `${balance} ETH`,
    });
  } else {
    await driver.findElement(balanceSelector);
  }
};
``

Reply:
make sense, i made the changes in this commit: https://github.com/MetaMask/metamask-extension/pull/27302/commits/74f65044e0cc40b508bd7ec569355bd47fd98400